docs/debugging.md

```diff
@@ -79,3 +79,53 @@ Open a gdb session -> start -> nop
 Done!
 
 ![pycharm9](https://github.com/hugsy/gef/assets/9882181/b22ec431-57e7-442a-835e-5817bdac7687)
+
+
+## Debugging with VSCode
+
+The approach to debug GEF with VSCode is relatively similar to that of PyCharm. Make sure to
+install the [Python extension for
+VSCode](https://marketplace.visualstudio.com/items?itemName=ms-python.python). This will install
+`debugpy`, a remote debugger that you can connect to from VSCode and debug anything in GEF from
+your session (breakpoints, watchpoints, etc.). Debugging a Python app from VSCode is [extensively
+ covered in the official docs](https://code.visualstudio.com/docs/python/debugging) to refer to
+ them if you're not sure how it works.
+
+To start a debugging session in GEF, manually run the following Python commands
+
+```python
+gef> pi import debugpy; debugpy.listen(5678); pi debugpy.wait_for_client()
+```
+
+Alternatively a convenience script named `vscode_debug.py` can also be found in the `scripts`
```

Comment:
very useful! 👍

Reply:
Ahah I've used this tiny snippet for years, never thought it was worth sharing before you suggested writing doc about debugging. Maybe we improve it later to integrate Pycharm (maybe other ides)